lib/views/create_project_form.dart

```diff
@@ -33,12 +35,32 @@ class _CreateProjectFormState extends State<CreateProjectForm> {
         crossAxisAlignment: CrossAxisAlignment.end,
         children: [
           fields(context),
-          LoadingButton(
-            text: "Create my project",
-            loading: isLoading,
-            onPressed: () {
-              submit();
-            },
+          LenraFlex(
+            spacing: 8,
+            children: [
+              LenraButton(
```

Comment:
Instead of making the same button having different reaction, I would have made two different buttons.
You should create them in the specific function.